build/windows/installer/livenessprobe/livenessprobe.cpp

```diff
@@ -133,5 +134,13 @@ int _tmain(int argc, wchar_t *argv[])
         return CERTIFICATE_RENEWAL_REQUIRED;
     }
 
+    if (argc > 5) {
```

Comment:
We would need to get the below yaml manifest change. what's the plan on getting this change rolled out first?

{{- if and .Values.OmsAgent.isUsingAADAuth .Values.OmsAgent.isWindowsAMAEnabled }}
                  - MonAgentCore.exe
{{- end }}

Reply:
Yes I have these changes in a branch. Will create a PR on this soon and try to get this next week itself, so we can release the new version in October